runtime/doc/message.txt

```diff
@@ -739,6 +739,12 @@ Example: >
 You tried to execute a command that is neither an Ex command nor
 a user-defined command.
 
+							*E905*  >
+  Cannot be changed after nvimrc files are sourced
```

Comment:
"nvimrc" is probably going away after XDG is adopted. Suggestion:

    Cannot set this option after startup.

"startup" is used in several areas of the Vim docs and is probably the most accurate and recognizable term.